tests/solver/test_lid_results.cpp

```diff
@@ -0,0 +1,1900 @@
+
```

Comment:
@michaeltryby some of these unit testing files don't have authors. do you want to have the consistency?

Reply:
@bemcdonnell I went ahead and updated all the headers for the test files.